app/views/spina/application/_mobile_navigation_items.html.haml

```diff
@@ -1,3 +1,3 @@
 %nav.d-flex.flex-column.flex-self-stretch
-  = render partial: 'mobile_navigation_item', collection: main_navigation_items, cached: true, as: :navigation_item
```

Comment:
I think this should instead be `-> navigation_item { [navigation_item, navigation_item.children] }` if you just need a composite cache key

Reply:
Thanks, didn't realise this was an option.